packages/bundle-source/test/test-bigint-transform.js

```diff
@@ -1,6 +1,6 @@
 // @ts-check
 import url from 'url';
-import { test } from './prepare-test-env-ava.js';
+import { test } from '@endo/ses-ava/prepare-test-env-ava.js';
```

Comment:
would be nice for it to match Ava:
```suggestion
import test from '@endo/ses-ava`;
```
The only existing export, `wrapTest`, is longer necessary.

Reply:
Removing `wrapTest` would be a breaking change to existing clients, especially agoric-sdk.

Reply:
Currently, this PR still leaves some endo uses of `wrapTest` in place. Most of these look like candidate to use the new import directly. I'll try in out is separate commits and see how it goes.

Reply:
It’s not necessary to remove `wrapTest` in order to also export a default `test`. I would like to keep the ball moving forward.

Reply:
Reminder that this weird trick is possible in the unlikely event it is necessary:
```js
import test, { wrapTest } from '@endo/ses-ava';
```

Reply:
DONE!

Turns out, JS supports a module have both named exports and a default export.

- All endo uses of `wrapTest` are now gone. 
- I left `wrapTest` in as a named export and commented it as deprecated (though not `@deprecated`). 
- I made `test` a default export of `@endo/ses-ava` 
- I removed the new package export `@endo/ses-ava/prepare-test-env-ava.js` and replaced all uses with
```js
import test from '@endo/ses-ava';
```
exactly as @turadg suggested.

***HOWEVER***, even leaving `wrapTest` in as a deprecated export, doing this does introduce a version slippage incompatibility:

Anyone doing 
```js
import { wrapTest } from '@endo/ses-ava';
```
had to do their own separate import to initialize ses and call `lockdown`. In order to export the new `test` from the same `index.js`, I had to make it do that initialization itself. That means that some existing clients will fail when these two initializations of ses collide.

Reply:
@kriskowal , I see you beat me to the punch ;)

But please see the ***HOWEVER*** in the immediately preceding comment. Advice appreciated.

Reply:
Ah, thank you for pointing out the caveat. I might have otherwise missed it.

I think we need a new entry-point for the convenience of importing a pre-wrapped `test`, both because of the backward-incompatibility you mention (which I’m sure will be caught with an integration branch check PR), but also because this precludes using `@endo/ses-ava` with any other combination of shims. Consider:

It might even be useful to have separate convenience entry-points for other layerings. That would let us continue to use `@endo/ses-ava` below the eventual-send layer.

```js
import test from '@endo/ses-ava/prepare-ses.js';
```

```js
import test from '@endo/ses-ava/prepare-endo.js';
```

This would allow folks to continue using `wrapTest` from `@endo/ses-ava` to compose other preparations at higher layers.

Reply:
Got it. I will reverse by bundling of the two into `index.js` and remove the deprecation comment on `wrapTest`.

`prepare-endo.js` seems like a fine name, even though it is still below the level of most of endo, since most of endo uses it.

Reply:
Separating it back out and renaming to prepare-endo.js, Done!